CMakeLists.txt

```diff
@@ -5,7 +5,7 @@ project(CppExtraSamples)
 set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${CMAKE_CURRENT_SOURCE_DIR}/cmake)
 include(CompilerOptions)
 
-set(ZIVID_VERSION 1.7.0)
+set(ZIVID_VERSION 1.8.0)
```

Comment:
@nedrebo, @SatjaSivcev 
This line is causing build to fail right now. Should I just remove it or will it get updated soon?

Reply:
The reason this fails is that the CI setup scrips must be updated to install 1.8.0 as well. I would do the bump to 1.8.0 in a separate PR to keep PRs and commits simple and separated.

However, the Zivid SDK is backwards compatible. Unless the samples use functionality introduced in 1.8.0 it is not necessary to bump the required version here.

Reply:
I see that you did the change in a separate commit, that is probably sufficient. No need for separate PR.

Reply:
Is it possible to make CI run on the previous commit and ignore this until it's updated?

Reply:
Yes, just don't do this change.